packages/libs/client/src/pact.ts

```diff
@@ -17,10 +17,12 @@ import {
   ISignatureJson,
   IUnsignedCommand,
   PactValue,
+  Type,
 } from '@kadena/types';
 
 import { IPactCommand } from './interfaces/IPactCommand';
 import { parseType } from './utils/parseType';
+import { ContCommand } from '.';
```

Comment:
As we're in the root, can we import directly from the file itself instead of from the barrel file?